demo/java/src/main/java/io/github/cosinekitty/astronomy/demo/Main.java

```diff
@@ -1,31 +1,28 @@
 package io.github.cosinekitty.astronomy.demo;
 
 import java.util.Date;
+
 import io.github.cosinekitty.astronomy.AstroTime;
 
 public class Main {
-    private static String UsageText = String.join(System.getProperty("line.separator"),
-        "Command line arguments:",
-        "",
-        "    now",
-        "    Display current date and time using AstroTime.",
-        ""
-    );
+    private static final String usageText = """
+            Command line arguments:
+
+                now
+                Display current date and time using AstroTime.
+            """;
 
     public static void main(String[] args) {
         int rc = 1;
         if (args.length == 0) {
-            System.out.println(UsageText);
+            System.out.println(usageText);
         } else {
             switch (args[0]) {
-                case "now":
+                case "now" -> {
```

Comment:
Oh, that is nice! I see the `switch` can be an expression, not a statement. In that case, I have an idea... I will do something fun here (I will keep you in suspense, LOL).

Reply:
Or maybe this is only in the newer Java also?

Reply:
Yes, it is for newer Java version, language features are fun, the great thing about Kotlin is its compiler is decoupled from JDK itself thus most of the feature are available even on oldest JDK versions.